lib/controllers/posts.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const Post = require('../models/Post');
+const authenticate = require('../middleware/authenticate');
+
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const data = await Post.getPosts();
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .post('/', authenticate, async (req, res, next) => {
+    try {
```

Comment:
To accomplish the limiting to 250 characters mentioned in the acceptance criteria on the assignment, you could (inside of this try block) use a variable to store the number of characters in the string passed through the detail on req.body, and if the characters exceed a limit, you could res.json a response noting the user needs to trim down their post.